Week_12/Ejercicio_1.py

```diff
@@ -0,0 +1,29 @@
+class BankAccount:
+    balance = 0
+
+    def add_money(self, amount):
+        self.balance += amount
+        print(f"Transaccion satisfactoria, el saldo actual de su cuenta es {self.balance}")
+
+    def subtract_money(self, amount):
+        if amount > self.balance:
+            print(f"Fondos insuficientes, usted dispone de {self.balance}")
+        else:
+            self.balance -= amount
+
+class SavingsAccount(BankAccount):
+    min_balance = 8000
```

Comment:
This should be configurable per instance. AKA, part of the _constructor_

Reply:
Implemented as part of the constructor method:
![image](https://github.com/JorgeA/DUAD/assets/4186222/3b9383ab-ca74-4326-a517-adf77011c44a)